libopenage/unit/resource.h

```diff
@@ -14,7 +14,8 @@ enum class game_resource {
 	wood,
 	food,
 	gold,
-	stone
+	stone,
+	fish
```

Comment:
This is supposed to be the resource stockpile types, so it shouldn't include fish (fishing should be added to the food stockpile)